sql/tabeles/challenges.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE `challenges` (
+ `id` int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
la primary keys cu auto increment in general ar fi bine sa fie unsigned
asta se aplica si la referintele aceleasi chei in alte tabele